intake/cli/client/subcommands/drivers.py

```diff
@@ -0,0 +1,79 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2018, Anaconda, Inc. and Intake contributors
+# All rights reserved.
+#
+# The full license is in the LICENSE file, distributed with this software.
+#-----------------------------------------------------------------------------
+'''
+
+'''
```

Comment:
Empty module documentation, is that on purpose ?

Reply:
Result of copy/paste from another CLI module. Updated, thanks.